src/RequestProcessor.cs

```diff
@@ -374,7 +376,21 @@ private StreamingMessage NewStreamingMessageTemplate(string requestId, Streaming
             InvocationRequest invocationRequest,
             FunctionInvocationPerformanceStopwatch stopwatch)
         {
-            throw new NotImplementedException(PowerShellWorkerStrings.DurableFunctionNotSupported);
+            if (!Utils.AreDurableFunctionsEnabled())
+            {
+                throw new NotImplementedException(PowerShellWorkerStrings.DurableFunctionNotSupported);
+            }
+
+            // Quote from https://docs.microsoft.com/en-us/azure/azure-functions/durable-functions-bindings:
```

Comment:
We should probably document this and make it available/visible to the user.

Reply:
Are you talking about documenting [this warning](https://docs.microsoft.com/en-us/azure/azure-functions/durable/durable-functions-bindings#trigger-behavior)?

Reply:
Yes. As long as the user knows where to get all the info required when working with Durable functions, then we should be good.